layouts/_default/index.html

```diff
@@ -21,7 +21,7 @@ <h1 class="post-title">
         </h1>
         <div class="post-meta">
           <span class="post-date">
-            {{ .Date.Format "2006-01-02" }}
+            {{ .Date.Format $.Site.Params.dateFormat }}
```

Comment:
This will break since not everyone has set `dateFormat` in their config, eg. people already have this theme installed. If you want to make it work and support older configs, please set here a default value:

```suggestion
            {{ .Date.Format ($.Site.Params.dateFormat | default "2006-01-02") }}
```